uni/lib/view/theme.dart

```diff
@@ -62,15 +62,15 @@ ThemeData applicationDarkTheme = ThemeData(
         secondary: _lightGrey,
         onSecondary: _darkishBlack,
         tertiary: _lightGrey,
-        onTertiary: _darkishBlack),
+        onTertiary: _lightGrey),
```

Comment:
onTertiary should contrast with the tertiary

Reply:
Right!